dev-client/package.json

```diff
@@ -46,7 +46,11 @@
     "react-native-vector-icons": "^10.0.2",
     "terraso-client-shared": "github:techmatters/terraso-client-shared#23fb61c",
     "uuid": "^9.0.1",
-    "yup": "^1.3.2"
+    "yup": "^1.3.2",
+    "expo": "^49.0.0",
+    "expo-screen-orientation": "~6.0.6",
+    "expo-sensors": "~12.3.0",
+    "expo-camera": "~13.4.4"
```

Comment:
question: we use `^` elsewhere. should we do the same here?

Reply:
`~` was also an auto generated by expo, it specifies a narrower range than `^` (`^` is a version range which means "this version and any patch or minor version updates to it", and `~` means "this version and any patch version updates to it"). more details [here](https://docs.npmjs.com/cli/v6/using-npm/semver#tilde-ranges-123-12-1). i'm not really sure without thinking further about it if one or the other is more appropriate for our needs, but i lean towards `^` for consistency absent any specific reason not to. thoughts?

Reply:
I would prefer `^` as well.